docs/specification/draft/basic/lifecycle.mdx

```diff
@@ -139,8 +146,9 @@ disconnect.
 <Note>
 If using HTTP, the client **MUST** include the `MCP-Protocol-Version:
 <protocol-version>` HTTP header on all subsequent requests to the MCP
-server.
-For details, see [the Protocol Version Header section in Transports](/specification/draft/basic/transports#protocol-version-header).
+server. For details, see [the Protocol Version Header section in Transports](/specification/draft/basic/transports#protocol-version-header).
+
+Similarly, if using WebSocket transport, the client **MUST** include an `mcp.version.${protocolVersion}` value in the WebSocket subprotocols list via the `Sec-WebSocket-Protocol` header in the WebSocket upgrade request.
```

Comment:
Again, we are doing this because of browser limitations. I think it's important to mention this, and show reference.